### DISCUSSION TITLE
Transition to AniList First App

### DISCUSSION THREAD

**Discussion by kl3jvi** [kl3jvi/animity#82] [Announcements]
Hello fellas!
It has been a long time since I announced a major update for the app and It's finally here. The app will now only work only with an AniList account and the only source of scraping anime as for now is gogoanime. So bare in mind that if an Anime is on anilist doesn't mean that it is on gogoanime so in this cases the app will show a blank page with 0 episodes for that anime until it goes live on gogoanime. The app has now a redone new UI and beautiful and smooth animations. The home screen was a little sluggish , but now it's smooth as a baby butt lol. I hope you guys engage and have some fun in the app. I am thinking of adding discussions inside app too. What do you think ?

**Comment by venkat2305** [comment]
why not add multiple providers.
